### PR TITLE
Remove backward compatability

### PR DESCRIPTION
**Pull request by LeighJoseph**
This change should remove core backward compatibility, I feel like it is unnecessary now that core fission is a thing. A quality of life change for both new and experienced players.

### CODE REVIEW

**Comment by l34Um1** [comment]
damn thanks for doing this

**Comment by DarkoniusXNG** [comment]
Can you update the tooltips for cores too? They are here: https://github.com/OpenAngelArena/oaa/blob/master/game/resource/English/ability/items/tooltip_cores.txt

**Comment by Larenall** [comment]
Who is this guy POG. Actuallly, who is this, seeing for the 1st time

**Comment by LeighJoseph** [comment]
It's Private Pigeon.

**Comment by Larenall** [comment]
> It's Private Pigeon.

Lmao

**Comment by DarkoniusXNG** [comment]
I forgot about this tooltip too: https://github.com/OpenAngelArena/oaa/blob/master/game/panorama/localization/addon_english.txt#L36

File: game/scripts/npc/items/item_aether_lens_3.txt [hunk 1]
```diff
@@ -17,9 +17,7 @@
     "AbilityTextureName"                                  "custom/recipe/recipe_3"
     "ItemRequirements"
     {
-      "01"                                                "item_aether_lens_2;item_upgrade_core_2;"
-      "02"                                                "item_aether_lens_2;item_upgrade_core_3;"
-      "03"                                                "item_aether_lens_2;item_upgrade_core_4;"
+      "01"                                                "item_aether_lens_2;item_upgrade_core_2;""
```
Comment (DarkoniusXNG) [inline game/scripts/npc/items/item_aether_lens_3.txt]: Travis fails at this line because of redundant "

**Pull request merged**